src/serverless-openapi-typescript.ts

```diff
@@ -183,8 +184,23 @@ export default class ServerlessOpenapiTypeScript {
         const outputFile = this.serverless.processedInput.options.output;
         const openApi = yaml.load(fs.readFileSync(outputFile));
         this.patchOpenApiVersion(openApi);
-        this.tagMethods(openApi);
-        fs.writeFileSync(outputFile, outputFile.endsWith('json') ? JSON.stringify(openApi, null, 2) : yaml.dump(openApi));
+        this.enrichMethodsInfo(openApi);
+        const encodedOpenAPI = this.encodeOpenApiToStandard(openApi);
+        fs.writeFileSync(outputFile, outputFile.endsWith('json') ? JSON.stringify(encodedOpenAPI, null, 2) : yaml.dump(encodedOpenAPI));
+    }
+
+    encodeOpenApiToStandard(openApi) {
+        const INVALID_CHARACTERS_KEY = /<|>/g;
+        const INVALID_CHARACTERS_REF = /%3C|%3E/g;
```

Comment:
1. I would add a comment about what we're doing here and why (how generics are generated with `>` and how these aren't legal character's in openapi 3.1 spec (with link)
2. `%3C` and `%3E` don't mean much - I would `atob('>')` to better capture the intent here, and also rename to `INVALID_CHARACTERS_ENCODED`

Reply:
8036b4b